src/model/include/iDynTree/ModelTransformers.h

```diff
@@ -65,6 +66,22 @@ bool createReducedModel(const Model& fullModel,
                         const std::vector<std::string>& jointsInReducedModel,
                         Model& reducedModel);
 
+/**
+ * This function takes in input a iDynTree::Model and
+ * an ordered list of joints and returns a model with
+ * just the joint specified in the list, with that exact order.
+ *
+ * All other joints are be removed by lumping (i.e. fusing together)
+ * the inertia of the links that are connected by that joint, assuming the joint
+ * to be in "rest" position (i.e. zero for revolute or prismatic joints), or the position
+ * specified in the removedJointPositions if a given joint is specified in removedJointPositions
+ *
+ */
+bool createReducedModel(const Model& fullModel,
+                        const std::vector<std::string>& jointsInReducedModel,
+                        Model& reducedModel,
+                        const std::map<std::string, double>& removedJointPositions);
```

Comment:
Is there a reason why you decided to use a map and not an unordered map?

Reply:
Because I am old and I am just instintively tend to use stuff that was available in old versions on C++? Anyhow, let's use unordered, I think it make sense.

Reply:
Fixed by https://github.com/robotology/idyntree/pull/1174/commits/36956df9fbe4a0fc8a0ab2ec3fa82ec7f0d4808b, let's see if CI is happy.